tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -202,7 +216,6 @@ export class GraphExecutor implements FunctionExecutor {
         names.map(name => this.graph.nodes[parseNodeName(name)[0]]);
     const outputNodeNames = outputs.map(name => parseNodeName(name)[0]);
     let outputNodes = outputNodeNames.map(name => this.graph.nodes[name]);
```

Comment:
Are there any cases where a user would want to collect intermediate tensors from multiple runs of the model? My fix above would make that very difficult because the user would have to clone the tensors again to avoid them being disposed on the next call to `model.execute`.

If this is an expected use case, maybe the model should have a public list of all the intermediate tensors for its past runs, and when `model.disposeIntermediateTensors` is called, it disposes all the tensors in the list.

To be clear, the list would have type `NamedTensorsMap[]`, not `Tensor[]`, and each entry would correspond to a run of the model.

Reply:
collect intermediate tensors from multiple runs is not supported by dump yet.

Reply:
collect intermediate tensors from multiple runs is not supported by dump yet.